ts/client/scripts/mainnet-no-margin-group.ts

```diff
@@ -0,0 +1,255 @@
+import { AnchorProvider, Wallet } from '@coral-xyz/anchor';
+import { Connection, Keypair, PublicKey } from '@solana/web3.js';
+import fs from 'fs';
+import { TokenIndex } from '../src/accounts/bank';
+import { Group } from '../src/accounts/group';
+import { MangoClient } from '../src/client';
+import { MANGO_V4_ID } from '../src/constants';
+import { toNative } from '../src/utils';
+
+const MAINNET_MINTS = new Map([
+  ['USDC', 'EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v'], // 0
+  ['SOL', 'So11111111111111111111111111111111111111112'], // 1
+]);
+const MAINNET_ORACLES = new Map([
+  ['USDC', 'Gnt27xtC473ZT2Mw5u8wZ68Z3gULkSTb5DuxJy7eJotD'],
+  ['SOL', 'H6ARHf6YXhGYeQfUzQNGk6rDNnLBQKrenN712K4AQJEG'],
+]);
+const MAINNET_SERUM3_MARKETS = new Map([
+  ['SOL/USDC', '8BnEgHoWFysVcuFFX7QztDmzuH8r5ZFvyP3sYwn1XTh6'],
+]);
+const {
+  MB_CLUSTER_URL,
+  MB_PAYER_KEYPAIR,
+  GROUP_NUM,
+}: {
+  MB_CLUSTER_URL: string;
+  MB_PAYER_KEYPAIR: string;
+  GROUP_NUM: number;
+} = process.env as any;
+
+const MIN_VAULT_TO_DEPOSITS_RATIO = 1;
+const NET_BORROWS_WINDOW_SIZE_TS = 24 * 60 * 60;
+const NET_BORROW_LIMIT_PER_WINDOW_QUOTE = toNative(0, 6).toNumber();
+
+const defaultOracleConfig = {
+  confFilter: 0.1,
+  maxStalenessSlots: null,
+};
+
+const defaultInterestRate = {
```

Comment:
All rates set to 0

Reply:
Since utilization will always be 0, rates should be 0 anyway. I think we need to check that the fast-listing instruction is usable? Could be, I think it sets reduce_only=2